examples/publication/frontend/best-practice-guidance.json

```diff
@@ -190,6 +190,236 @@
         "links": {
         }
       }
+    ],
+    "secondary_to_step_by_steps": [
+      {
+        "title": "Learn to drive a car: step by step",
+        "content_id": "e01e924b-9c7c-4c71-8241-66a575c2f61f",
+        "publishing_app": "collections-publisher",
+        "rendering_app": "collections",
+        "locale": "en",
+        "updated_at": "2018-01-18T15:46:10.000+00:00",
+        "public_updated_at": "2018-01-18T15:46:10.000+00:00",
+        "document_type": "step_by_step_nav",
+        "schema_name": "step_by_step_nav",
+        "base_path": "/learn-to-drive-a-car",
+        "description": "Learn to drive a car in the UK - get a provisional licence, take driving lessons, prepare for your theory test, book your practical test.",
+        "links": {
+          "pages_secondary_to_step_nav": [
+            {
+              "api_path": "/api/content/government/publications/car-and-small-van-driving-syllabus",
+              "base_path": "/government/publications/car-and-small-van-driving-syllabus",
+              "content_id": "5e5dc201-7631-11e4-a3cb-005056011aef",
+              "document_type": "guidance",
+              "locale": "en",
+              "public_updated_at": "2014-11-24T15:00:02.000+00:00",
+              "schema_name": "publication",
+              "title": "Learning to drive a car syllabus",
+              "withdrawn": false
```

Comment:
I think the only thing that's missing from this example is an empty `links` hash here, similar to: https://www.gov.uk/api/content/apply-first-provisional-driving-licence